crates/ide-db/src/prime_caches.rs

```diff
@@ -108,10 +108,9 @@ pub fn parallel_prime_caches(
                     hir::attach_db(&db, || {
                         // method resolution is likely to hit all trait impls at some point
                         // we pre-populate it here as this will hit a lot of parses ...
-                        _ = hir::TraitImpls::for_crate(&db, crate_id);
-                        // we compute the lang items here as the work for them is also highly recursive and will be trigger by the module symbols query
+                        // This also computes the lang items, which is what we want as the work for them is also highly recursive and will be trigger by the module symbols query
                         // slowing down leaf crate analysis tremendously as we go back to being blocked on a single thread
-                        _ = hir::crate_lang_items(&db, crate_id);
```

Comment:
Turns out this was unnecessary to be added here, its the first query that `TraitImpls::for_crate` runs anyways